conf/config_utils.py

```diff
@@ -7,16 +7,12 @@
 
 
 def from_file(fname):
-    if not os.path.exists(fname):
-        log.warning('config file does not exist: "{}"'.format(fname))
-        return {}
     try:
         with open(fname) as buf:
             conf = json.load(buf)
             return conf
-    except Exception:
-        log.warning('Error loading config file', exc_info=1)
```

Comment:
adding `fname` to this warning would probably fix the problems raised by @leophys